src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -369,7 +369,9 @@ describe('Test suite for dictionary concepts components', () => {
     dictionaryConceptsWrapper.forceUpdate();
     wrapper.find('#CIEL').simulate('change', sourceEvent);
     wrapper.find('#Diagnosis').simulate('change', classesEvent);
-    expect(spy).toHaveBeenCalledTimes(2);
+    classesEvent.target.type = undefined;
+    wrapper.find('#Diagnosis').simulate('change', classesEvent);
```

Comment:
Why are we duplicating the above line?

Reply:
The second call to `simulate` takes in an undefined target type and therefore different from the first.